libindy-crypto/src/cl/prover.rs

```diff
@@ -60,14 +60,14 @@ impl Prover {
             ur: blinded_revocation_master_secret.as_ref().map(|d| d.ur)
         };
 
-        let master_secret_blinding_data = MasterSecretBlindingData {
+        let master_secret_blinding_factor = MasterSecretBlindingData {
```

Comment:
I don't like renaming of these variables before we rename the structure. It just decreases code readability.

Reply:
The reason for not changing `MasterSecretBlindingData` is because that would require changing wrappers and people might have code built against it, that is the reason i had added a TODO over `MasterSecretBlindingData`